lib/SlackGhost.js

```diff
@@ -151,7 +152,7 @@ SlackGhost.prototype.uploadContent = function(file) {
     }).then((response) => {
         var content_uri = JSON.parse(response).content_uri;
 
-        console.log("Media uploaded to " + content_uri);
+        log.debug("Media uploaded to " + content_uri);
```

Comment:
info? Feels a bit less debug-y, although could really be argued either way

Reply:
I'm just going to leave this on debug on the basis that it's just telling us media got uploaded, which is what we would expect to happen and it doesn't really tell us a whole lot unless we are debugging an issue.